src/ticks/syms.c

```diff
@@ -122,10 +124,32 @@ void read_symbol_file(char *filename)
                     } else {
                         sym->symtype = SYM_ADDRESS;
                     }
+
+                    if(argc >= 8) {
+                      length = strlen(sym->name) + strlen(argv[7]);
+                      temp = malloc(length + 2 /* '_' and null terminator */ );
+                      sprintf(temp, "%s_%s", sym->name, argv[7]);
```

Comment:
Can you replace that for utstring? Plenty of examples of it within the project.

Reply:
Ok. I've updated these changes to use utstring now